File: FirmaTransport/FirmaTransport/domain/Oficiu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FirmaTransport.domain
{
    class Oficiu : Entity<int>
    {
        public string nume { get; set; }
        public string parola { get; set; }

        public Oficiu(int id, string nume, string parola)
        {
            this.id = id;
            this.nume = nume;
            this.parola = parola;
        }

        public override string ToString()
        {
            return id + " " + nume +" "+ parola ;
        }
    }
}

File: FirmaTransportSocketsJava/Model/src/main/java/domain/Oficiu.java
package domain;

public class Oficiu extends Entity<Integer> {
    private String nume;
    private String parola;

    public Oficiu(String nume, String parola) {
        this.nume = nume;
        this.parola=parola;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getParola() {
        return parola;
    }

    public void setParola(String parola) {
        this.parola = parola;
    }

    @Override
    public String toString() {
        return "Oficiu{" +
                "nume='" + nume + '\'' +
                ", parola='" + parola + '\'' +
                ", id=" + id +
                '}';
    }
}
